src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -127,13 +143,24 @@ neg-involutive p rewrite neg-involutive-≡ p = ≃-refl
   1ℤ ℤ.* (↥(- q) ℤ.* ↧ p)   ≡⟨ ℤ.*-identityˡ (↥(- q) ℤ.* ↧ p) ⟩
   ↥(- q) ℤ.* ↧ p            ∎ where open ≡-Reasoning
 
-neg-mono-<-> : -_ Preserves  _<_ ⟶ _>_
-neg-mono-<-> {p} {q} (*<* p<q) = *<* $ begin-strict
+neg-mono-< : -_ Preserves  _<_ ⟶ _>_
+neg-mono-< {p} {q} (*<* p<q) = *<* $ begin-strict
   ℤ.-  ↥ q ℤ.* ↧ p    ≡˘⟨ ℤ.neg-distribˡ-* (↥ q) (↧ p) ⟩
   ℤ.- (↥ q ℤ.* ↧ p)   <⟨ ℤ.neg-mono-< p<q ⟩
   ℤ.- (↥ p ℤ.* ↧ q)   ≡⟨ ℤ.neg-distribˡ-* (↥ p) (↧ q) ⟩
   ↥ (- p) ℤ.* ↧ (- q) ∎ where open ℤ.≤-Reasoning
```

Comment:
`where` should be on the next line

Reply:
Should I change `-‿cong` as well (it's previous definition which uses this style).

Reply:
That'd be great if you were willing to.